src/set_theory/surreal.lean

```diff
@@ -253,9 +253,17 @@ theorem numeric_add : Π {x y : pgame} (ox : numeric x) (oy : numeric y), numeri
  end⟩
 using_well_founded { dec_tac := pgame_wf_tac }
 
--- TODO prove
--- theorem numeric_nat (n : ℕ) : numeric n := sorry
--- theorem numeric_omega : numeric omega := sorry
+/-- Pre-games defined by natural numbers are numeric. -/
+theorem numeric_nat (n : ℕ) : numeric n :=
+begin
+  induction n with n hn,
+  apply numeric_zero,
+  apply numeric_add hn numeric_one,
+end
```

Comment:
Does this work as follows?
```suggestion
theorem numeric_nat : Π (n : ℕ), numeric n
| 0 := numeric_zero,
| (n+1) := numeric_add (numeric_nat n) numeric_one
```

Reply:
Yes it does! (without the comma)
I'll make the change. 
Thanks,

Reply:
Done.